lib/pages/detail/detail_content.dart

```diff
@@ -239,4 +243,63 @@ extension on Company {
       return null;
     }
   }
+
+  CompanyScoreData? scoreData() {
+    final int? plCapital = this.plCapital;
+    final int? plWorkers = this.plWorkers;
+    final int? plRnD = this.plRnD;
+    final int? plRegistered = this.plRegistered;
+    final int? plNotGlobEnt = this.plNotGlobEnt;
+    final int? plScore = this.plScore;
+
+    if (plCapital != null &&
+        plWorkers != null &&
+        plRnD != null &&
+        plRegistered != null &&
+        plNotGlobEnt != null &&
+        plScore != null) {
+      return CompanyScoreData(
+          plCapital: plCapital.toDouble(),
+          plWorkers: plWorkers != 0,
+          plRnD: plRnD != 0,
+          plRegistered: plRegistered != 0,
+          plNotGlobEnt: plNotGlobEnt != 0,
+          plScore: plScore);
+    }
+  }
+}
+
+class _ScoreSection extends StatelessWidget {
+  final Company company;
+
+  const _ScoreSection({super.key, required this.company});
+
+  @override
+  Widget build(BuildContext context) {
+    final scoreData = company.scoreData();
+
+    // Check if scoreData is not null and if the score is zero
+    if (scoreData != null) {
+      if (scoreData.plScore == 0) {
```

Comment:
Po co ten if na `scoreData.plScore == 0`? Warunkiem na pokazanie informacji o niezweryfikowanej firmie jest brak scoreData. Jeśli scoreData jest to zwracamy  `CompanyScoreWidget`. Jeśli jej nie ma to zwracamy `NoScoreMessage()`. Linijki od 283 do 299 są do usunięcia. Tam gdzie zwracamy `Container` ma `NoScoreMessage`

Reply:
Usunąłeś zawartość if, ale sam if też do wywalenia. Linia 185 i 186 do kosza.